### ISSUE TITLE
Output diagnostic plots not consistent font size

### ISSUE THREAD

**Issue by ggalibert** [aodn/imos-toolbox#373]
The diagnostic / report plots output by the toolbox do not have a consistent font size across data providers. It is directly dependent on the data provider's operating system configuration and can lead to messy legends.

Plot generated below by Windows64:
![imos_anmn-nrs_nrskai_fv01_nrskai-1308_line_depth_c-20160922t070249z_windows](https://cloud.githubusercontent.com/assets/3498686/18739693/ad50afce-80e8-11e6-9c65-4612aa845068.png)

Same plot generated by Linux64:
![imos_anmn-qld_nrskai_fv01_nrskai-1308_line_depth_c-20160922t064258z](https://cloud.githubusercontent.com/assets/3498686/18739736/09a44c9a-80e9-11e6-8a51-3d29040d45bf.png)

**Comment by ggalibert** [comment]
* One part of the problem is that pixels are not the same from one config to another. On my Windows:
```Matlab
ScreenPixelsPerInch = [90]
```
While on my Linux:
```Matlab
ScreenPixelsPerInch = [96]
```

* Another problem is the default font that is adopted by Matlab. On both my Windows and Linux:
```Matlab
>> get(0,'defaultAxesFontName')

ans =

Helvetica

>> get(0,'defaultTextFontName')

ans =

Helvetica
```

However, these two fonts don't exist on both my windows and linux configurations! Two other fonts are then chosen that are distinct from one OS to the other.

**Comment by ggalibert** [comment]
The list of font that Matlab can render on your OS on a figure can be obtained by calling ```uisetfont```.

**Comment by ggalibert** [comment]
The problems listed above are smaller compared to the one that the figure printed via the GUI is different from the one printed using commands ```print``` or ```imwrite```.
![windows-sansserif-manual](https://cloud.githubusercontent.com/assets/3498686/18824017/78a3f3c2-8402-11e6-846e-c7e86e1ba5ad.png)
![windows-sansserif-auto](https://cloud.githubusercontent.com/assets/3498686/18824020/7c6e9e1c-8402-11e6-9608-b000dbe7c342.png)

**Comment by ggalibert** [comment]
See "simulate PRINT command" in https://au.mathworks.com/matlabcentral/answers/56433-is-there-a-computationally-fast-way-to-save-figures-as-pictures . Solves part of the issue.

**Comment by ggalibert** [comment]
Finally decided to use [ScreenCapture](http://au.mathworks.com/matlabcentral/fileexchange/24323-screencapture-get-a-screen-capture-of-a-figure-frame-or-component) from Yair Altman to deal with inconsistencies between what you see and what you print. The main drawback is that the size of the plots are not consistent across data providers and depends on their PC screen resolution. At least it is neat and readable.

**Comment by ggalibert** [comment]
Commits https://github.com/aodn/imos-toolbox/commit/39791f3ed29cf9df59a5b26b6fdefc03ed9ba28e and https://github.com/aodn/imos-toolbox/commit/95acf6b7d26ca6f49dc2ca8c08fc6cf50dd051ca fix the issue in [v2.5.17](https://github.com/aodn/imos-toolbox/releases/tag/v2.5.17)

**Comment by ggalibert** [comment]
Results:
![imos_anmn-qld_gbrcch_fv01_gbrcch-1103_line_cphl_c-20161027t053159z](https://cloud.githubusercontent.com/assets/3498686/19756046/b72e277c-9c65-11e6-901a-a862f3ef5fdc.png)
